ReactWindows/ReactNative/Views/TextInput/ReactTextInputShadowNode.cs

```diff
@@ -14,7 +15,7 @@
 namespace ReactNative.Views.TextInput
 {
     /// <summary>
-    /// This extension of <see cref="LayoutShadowNode"/> is responsible for 
+    /// This extension of <see cref="LayoutShadowNode"/> is responsible for
     /// measuring the layout for Native <see cref="TextBox"/>.
     /// </summary>
     public class ReactTextInputShadowNode : LayoutShadowNode
```

Comment:
>ReactTextInputShadowNode [](start = 17, length = 24)

While you're here, what about border thickness?  That's something that would definitely affect the measurement...